### PR TITLE
use Play WS API instead of akka-http

### PR DESCRIPTION
**Pull request by novakov-alexey**

### CODE REVIEW

**Comment by DmytroOrlov** [comment]
```
[error] Failed tests:
[error] 	controllers.OneAppSpecs
```

**Comment by novakov-alexey** [comment]
Travis CI status is green

File: build.sbt [hunk 1]
```diff
@@ -42,24 +42,23 @@ lazy val server = (project in file("server"))
       (baseDirectory.value / "test" / "tests" * "*").get map
         (x => x -> ("test/tests/" + x.getName)),
 
+    resourceDirectory in Test := baseDirectory.value / "test-resources",
```
Comment (DmytroOrlov) [inline build.sbt]: I think we should not customize everything, remove this line and
```
mv server/test-resources/ server/test/resources/
```

**Comment by DmytroOrlov** [comment]
```
$ cat .travis.yml 
language: scala
scala:
   - 2.11.8
jdk:
  - oraclejdk8
script: sbt ++$TRAVIS_SCALA_VERSION "testOnly -- -l RequireDB"
```

> Travis CI status is green

sure!

File: build.sbt (hunk 1 above)
Comment (novakov-alexey) [inline build.sbt]: we have scala packages in server/test. Do you propose to add resources as package in test?

Comment (DmytroOrlov) [inline build.sbt]: No, just remove your customization and use defaults.

Comment (novakov-alexey) [inline build.sbt]: Do we need to create additional folder in the middle: "test / scala / test" as per standard maven template here?

Comment (DmytroOrlov) [inline build.sbt]: Let's use maven-style or play-style, but not our custom.

**Comment by novakov-alexey** [comment]
Fixed!

File: server/test/controllers/GitHubUserTest.scala [hunk 2]
```diff
@@ -1,35 +1,46 @@
...
+import play.api.libs.ws.{WSClient, WSResponse}
 import play.api.mvc.Result
 import play.api.test.FakeRequest
 import play.api.test.Helpers._
 
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.Future
 
-@DoNotDiscover class GitHubUserTest extends PlaySpec with ConfiguredApp {
+@DoNotDiscover class GitHubUserTest extends PlaySpec with ConfiguredApp with MockFactory {
```
Comment (DmytroOrlov) [inline server/test/controllers/GitHubUserTest.scala]: `@DoNotDiscover` makes me cry when I want to run just one test or this **Spec** file

File: server/test/controllers/GitHubUserTest.scala [hunk 3]
```diff
@@ -1,35 +1,46 @@
...
   private implicit lazy val s = app.actorSystem
   private implicit lazy val m = app.materializer
+  private implicit lazy val ws = mock[WSClient]
```
Comment (DmytroOrlov) [inline server/test/controllers/GitHubUserTest.scala]: https://www.playframework.com/documentation/2.5.x/ScalaFunctionalTestingWithScalaTest#testing-ws-calls
There are more test cases :)

**Pull request merged**